ios/VizbeeNativeManager.m

```diff
@@ -203,6 +203,32 @@ -(void)sendEvent:(NSString*) name
     [[Vizbee getSessionManager] disconnectSession];
 }
 
+//----------------
+#pragma mark - Signin APIs
+//----------------
+
+RCT_EXPORT_METHOD(onGetSignInInfo:(NSDictionary*) signInInfo) {
+    RCTLogInfo(@"Invoking onGetSignInInfo");
```

Comment:
Let's follow the updated logging format you will get it after rebasing with the main.

Reply:
Updated the logging to new format